test/rules.test.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs')
+const path = require('path')
+const globby = require('globby')
+const fastmatter = require('fastmatter')
+const marked = require('marked')
+
+const rulesPaths = globby.sync([`./_rules/*.md`])
+const rulesData = rulesPaths.map(rulePath => {
+  const filename = path.parse(rulePath).base
+  const fileContents = fs.readFileSync(rulePath, { encoding: 'utf-8' })
+  const { attributes, body } = fastmatter(fileContents)
```

Comment:
Rename `attributes` to `frontmatter` straight away, rather than have this same variable exist under two names in the code.